airflow/cli/cli_parser.py

```diff
@@ -629,41 +629,41 @@ def add_to_parser(self, parser: argparse.ArgumentParser):
     default=False)
 # users
 ARG_USERNAME = Arg(
-    ('--username',),
+    ('--username', '-u'),
```

Comment:
Let's change the order the other way around, short-option followed by long

```suggestion
    ('-u', '--username'),
```

Current tests are also failing because of it :

```
tests/cli/test_cli_parser.py .F

____________________ TestCli.test_arg_option_mix_short_long ____________________

self = <tests.cli.test_cli_parser.TestCli testMethod=test_arg_option_mix_short_long>

    def test_arg_option_mix_short_long(self):
        """
        Test if the name of cli.args mix option (-s, --long) valid
        """
        optional_mix = [
            arg
            for arg in cli_args.values()
            if len(arg.flags) == 2 and arg.flags[0].startswith("-")
        ]
        for arg in optional_mix:
            self.assertIsNotNone(LEGAL_SHORT_OPTION_PATTERN.match(arg.flags[0]),
>                                f"{arg.flags[0]} is not match")
E           AssertionError: unexpectedly None : --username is not match

tests/cli/test_cli_parser.py:63: AssertionError
```

Reply:
Whoops, will fix